composer/metrics/nlp.py

```diff
@@ -542,6 +543,14 @@ def get_client(self) -> EvalClient:
                 'CODE_EVAL_DEVICE to LOCAL or LAMBDA.')
         return client
 
+    def estimator(self, n: int, c: int, k: int) -> float:
+        """
+        Calculates 1 - comb(n - c, k) / comb(n, k).
```

Comment:
Can you please add a comment linking to the humaneval paper to justify this estimator?

Reply:
done